webpack.config.base.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
 
+var _ = require('lodash');
+var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
   module: {
-    loaders: [
-      { test: /\.js$/, loaders: ['babel-loader'], exclude: /node_modules/ }
-    ]
+    loaders: [{
+      test: /\.js$/,
+      loaders: ['babel-loader'],
+      include: [
+        path.resolve(__dirname, 'src'),
+        path.resolve(__dirname, 'node_modules', 'lodash')
+      ]
+    }]
+  },
+  node: {
+    process: false
   },
   output: {
     library: 'Redux',
     libraryTarget: 'umd'
   },
-  resolve: {
-    extensions: ['', '.js']
-  }
+  plugins: [
+    ['lodash/internal/baseIteratee.js', './toFunction.js'],
```

Comment:
The biggest reduction in size is from replacing `baseInteratee` references with `toFunction` to cut out the callback shorthand helpers (e.g. `_.mapValues(objects, 'propName')`).